rcljava/src/main/java/org/ros2/rcljava/parameters/service/ParameterService.java

```diff
@@ -15,4 +15,5 @@
 
 package org.ros2.rcljava.parameters.service;
 
+// TODO(ivanpauno): What is this interface for?
```

Comment:
I'm not sure. My only guess is to have a place to add future API that may be common to parameter service implementations.

Reply:
Yeah, it sounds strange to me.
I've deleted the TODO, I can ask again when opening PRs upstream.